runtime/commonMain/src/kotlinx/serialization/SerialDescriptor.kt

```diff
@@ -176,9 +168,6 @@ public interface SerialDescriptor {
      */
     public val annotations: List<Annotation> get() = emptyList()
 
-    @Deprecated(message = "Deprecated in the favour of 'annotations' property", replaceWith = ReplaceWith("annotations"))
```

Comment:
DeprecationLeval.ERROR instead of removing?

Reply:
Later removal of this method affects both API and ABI, so everyone should have already migrated from it